File: Prefix Sum/1588. Sum of All Odd Length Subarrays.cpp
class Solution {
public:
    int sumOddLengthSubarrays(vector<int>& arr) {
      int ans = 0;
        for(int i=0;i<arr.size();i++) {
            int cnt=(arr.size()-i)*(i+1);
            cnt = cnt/2+cnt%2;
            ans+=cnt*arr[i];
        }
        return ans;  
    }
};

File: Prefix Sum/1588. Sum of All Odd Length Subarrays.js
var sumOddLengthSubarrays = function(arr) {
    
    var i;
    var sum=0;
    for(i=0;i<arr.length;i++)
        {
            var count=(arr.length-i)*(i+1)
            count =Math.floor(count/2)+count%2;
            sum+=(arr[i]*count);
        }
    return sum;
    
};
